examples/random_forest/utils/prepare_data_split.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Can the "random_forest" be put inside "examples/xgboost/tree-based"?

By that we just reuse the script "prepare_data_split.py" from "examples/xgboost"?

Reply:
same for some readme section and prepare_job_config.py

Reply:
given that random forest and boosting are two algorithms, we may keep them separate. Although they can be implemented with the same underlying library of xgboost

Reply:
data split part can further follow sklearn examples - future refactor